Source/gg2/Scripts/Events/doEventPlayerDeath.gml

```diff
@@ -218,6 +218,13 @@ if (hasReward(victim, 'Ghost') and victim.ghost == noone) {
     victim.ghost.vspeed = vspeed;
 }
 
+if (hasClassReward(victim, 'Tombstone'))

```

Comment:
Why are we treating the tombstones different from the statues? IMO they are so similar in concept that they might as well behave the same way.